types/index.d.ts

```diff
@@ -97,7 +97,11 @@ interface DownloadEvents {
 type FilenameCallback = (fileName: string, filePath: string) => string;
 interface FilenameDefinition {
   name: string;
-  ext: string;
+  /** The extension of the file. It may be a boolean: `true` will use the `name` property as the full file name (including the extension),
+  `false` will keep the extension of the downloaded file.
+  
+  (default:false) */
+  ext?: string | boolean;
```

Comment:
Does `true` do anything here?
```suggestion
  ext?: string | false;
```